### DISCUSSION TITLE
Blog Revision Module

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#784] [General]
# Blog Revision Module

Show revision information below the post content.

https://hbstack.dev/modules/blog/revision/

**Comment by FuadEfendi** [comment]
it is hard to find how to configure it; I found only "post_date_format" in source code

**Reply by FuadEfendi** [reply]
ops I found it; https://hbstack.dev/modules/blog/overview/

**Reply by razonyang** [reply]
> it is hard to find how to configure it; I found only "post_date_format" in source code

Indeed, I'm planning to make parameters as headings, so that you can find it through the search form (but it takes time, may not happen fast).
